README.md

```diff
@@ -90,7 +90,7 @@ For more models, go to [**tensorflow/swift-models**][swift-models].
 ### Requirements
 
 * [Swift for TensorFlow toolchain][toolchain].
-* An environment that can run the Swift for TensorFlow toolchains: Linux 18.04 or macOS with Xcode 10.
+* An environment that can run the Swift for TensorFlow toolchains: Ubuntu 18.04, macOS with Xcode 10, or Windows 10.
```

Comment:
@compnerd: is Windows 10 required for building `tensorflow/swift-apis`?
I didn't see it mentioned in [Windows.md](https://github.com/apple/swift/blob/master/docs/Windows.md) or [WindowsBuild.md](https://github.com/apple/swift/blob/master/docs/WindowsBuild.md).

Reply:
It is mentioned here: https://github.com/tensorflow/swift/blob/master/Installation.md#windows , but it can also be wrong.

Reply:
Yeah, Windows 10 is the only supported windows version.  Although it is possible to support older windows, windows 8 is already at the end of mainstream support.